docker-compose.yml

```diff
@@ -0,0 +1,211 @@
+version: "3.9"
+
+services:
+  # contracts
+
+  poa-ganache:
+    image: trufflesuite/ganache:v7.1.0
+    command: --wallet.deterministic --wallet.totalAccounts 50
+    ports:
+      - "8546:8545"
+    healthcheck:
+      test: ["CMD", "node", "-e", "require('http').request('http://localhost:8545').end()"]
+      interval: 5s
+      timeout: 5s
+    deploy:
+      mode: global
+
+  poa-deploy-contracts:
+    build: contracts
+    command: npx truffle migrate --to 4 --network audius_dev
+    depends_on:
+      poa-ganache:
+        condition: service_healthy
+
+  # eth-contracts
+
+  eth-ganache:
+    image: trufflesuite/ganache:v7.1.0
+    command: --wallet.deterministic --wallet.totalAccounts 50 --miner.blockTime 1
+    ports:
+      - "8545:8545"
+    healthcheck:
+      test: ["CMD", "node", "-e", "require('http').request('http://localhost:8545').end()"]
+      interval: 5s
+      timeout: 5s
+    deploy:
+      mode: global
+
+  eth-blockscout-db:
+    image: postgres:13.6
+    environment:
+      POSTGRES_PASSWORD: 'postgres'
+      POSTGRES_USER: 'postgres'
+    healthcheck:
+      test: ["CMD", "pg_isready"]
+      interval: 10s
+      timeout: 5s
+    deploy:
+      mode: global
+
+  eth-blockscout:
+    image: blockscout/blockscout:latest
+    command: 'mix do ecto.create, ecto.migrate, phx.server'
+    environment:
+      ETHEREUM_JSONRPC_VARIANT: 'ganache'
+      ETHEREUM_JSONRPC_HTTP_URL: 'http://eth-ganache:8545'
+      ETHEREUM_JSONRPC_WS_URL: 'ws://eth-ganache:8545'
+      INDEXER_DISABLE_PENDING_TRANSACTIONS_FETCHER: 'true'
+      DATABASE_URL: 'postgresql://postgres:postgres@eth-blockscout-db:5432/postgres?ssl=false'
+      ECTO_USE_SSL: false
+    ports:
+      - "9545:4000"
+    depends_on:
+      eth-blockscout-db:
+        condition: service_healthy
+      eth-ganache:
+        condition: service_healthy
+
+  eth-deploy-contracts:
+    build: eth-contracts
+    command: npx truffle migrate --to 12 --network audius_dev
+    depends_on:
+      eth-ganache:
+        condition: service_healthy
+
+  # solana-programs
+
+  solana-test-validator:
+    image: solanalabs/solana:v1.10.11
```

Comment:
@jowlee to confirm these images - i think we no longer want to use this image

Reply:
I think we should have the option to run it, but the default used container should be the pre-deployed to reduce the amount of time during A up

Reply:
I agree, I am working on this right now